src/Summoner/License.hs

```diff
@@ -15,25 +18,47 @@ import qualified Data.Text as T
 -- License
 ----------------------------------------------------------------------------
 
-licenseNames :: [License]
-licenseNames = map fst githubLicenseQueryNames
-
-githubLicenseQueryNames :: [(License, Text)]
-githubLicenseQueryNames =
-    [ ("MIT",        "mit")
-    , ("BSD2",       "bsd-2-clause")
-    , ("BSD3",       "bsd-3-clause")
-    , ("GPL-2",      "gpl-2.0")
-    , ("GPL-3",      "gpl-3.0")
-    , ("LGPL-2.1",   "lgpl-2.1")
-    , ("LGPL-3",     "lgpl-3.0")
-    , ("AGPL-3",     "agpl-3.0")
-    , ("Apache-2.0", "apache-2.0")
-    , ("MPL-2.0",    "mpl-2.0")
-    ]
-
-newtype License = License { unLicense :: Text }
-    deriving (IsString, Eq, Ord, Show)
+data LicenseName = MIT
```

Comment:
Sorry, I'm not sure if we have this in our style guide, but I recently using this kind of formatting everywhere, How do you like that?

```haskell
data LicenceName
    = MIT
    | BSD2
    | ...
    deriving (...)
```

Reply:
Yes, It is better. How I missed it!

Reply:
@vrom911 Can this rule be added to [`.stylish-haskell`](https://github.com/kowainik/summoner/blob/master/.stylish-haskell.yaml)? I'm not sure if it's supported. :thinking:

Reply:
That would be very nice, but I don't see in the docs that it's capable to do that 😞